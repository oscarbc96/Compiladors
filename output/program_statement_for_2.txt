1: $t01 := 2
2: b := $t01
3: a := b
4: IF a LEI 4 GOTO 6
5: GOTO 15
6: PARAM 123
7: CALL PUTI,1
8: GOTO 4
9: PARAM 123
10: CALL PUTI,1
11: PARAM 123
12: CALL PUTI,1
13: a := a ADDI 1
14: GOTO 4
15: HALT
