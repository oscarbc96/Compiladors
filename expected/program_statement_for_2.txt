1: $t01 := 2
2: b := $t01
3: a := b
4: IF a LTI 4 GOTO 6
5: GOTO 16
6: PARAM 123
7: CALL PUTI,1
8: GOTO 4
9: PARAM 123
10: CALL PUTI,1
11: PARAM 123
12: CALL PUTI,1
13: $t02 := a ADDI 1
14: a := $t02
15: GOTO 4
16: HALT
